datetime_test.go

```diff
@@ -2,11 +2,13 @@ package faker
 
 import (
 	"fmt"
-	"github.com/bxcodec/faker/support/slice"
 	"log"
 	"reflect"
 	"testing"
 	"time"
+
+	"github.com/bxcodec/faker/support/slice"
+	"github.com/stretchr/testify/assert"
```

Comment:
I know `testify` is really helpful. But I would recommend to not using it here. I'd prefer if we just using a  simple assertion manually like other test-case. It's just to avoid any possible future conflict related to dependencies because we don't use any dependency management tools here (Dep, Glide etc).